docs/content/documentation/content/table-of-contents.md

```diff
@@ -34,3 +34,20 @@ While headers are neatly ordered in this example, it will work just as well with
 
 Note that all existing HTML tags from the title will NOT be present in the table of contents to
 avoid various issues.
+
+## Conditional rendering
```

Comment:
I think it would be clearer if you just wrapped the example above in `{% if page.toc %}`